sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/CpkScopeInfo.java

```diff
@@ -11,9 +11,9 @@
 /**
  * Additional parameters for a set of operations.
  */
-@JacksonXmlRootElement(localName = "encryption-scope")
+@JacksonXmlRootElement(localName = "cpk-scope-info")
```

Comment:
Was this element name supposed to change?

Reply:
I originally had a transform that made it encryption scope instead of cpk scope info

Reply:
that name itself does not matter